src/components/templates/EntranceExperience/EntranceExperience.tsx

```diff
@@ -270,6 +273,21 @@ export const EntranceExperience: React.FunctionComponent<EntranceExperienceProps
                   })}
                 </>
               )}
+            {isUserVenueOwner && (
+              <InformationCard title="Enter the venue as an admin">
```

Comment:
Not keen on this copy. Should be something like "Preview what an event looks like in your venue". Please confirm text with Mathieu Robert.